### DISCUSSION TITLE
Fullstack .Net | Identity no Blazor -  Autenticação via cookie

### DISCUSSION THREAD

**Discussion by edunardi** [balta-io/forum#628] [Tira Dúvidas] [answered]
Olá, durante o curso pensei em uma situação, se o endpoint da API estivesse disponível através de um load balance por exemplo e ela estivesse publicada em vários servers, a autenticação via cookie funcionaria numa boa nesse cenário? independente de qual servidor respondesse ou seria necessário uma autenticação através de token mesmo?

**Comment by andrebaltieri** [comment, marked as answer]
No caso da implementação base, o Cookie é gerado pelo identity com base na aplicação, então, como todas as máquinas tem a mesma aplicação, é tranquilo.

Agora se você quiser armazenar este Cookie para invalidá-lo mais tarde, ai tem que pensar nisso, não pode armazenar na memória nem no storage local da máquina, precisa ser compartilhado entre as máquinas.
